### ISSUE TITLE
When we upload a new APK to production can we reuse the previous expansion file? 

### ISSUE THREAD

**Issue by mirzabros** [moust/phonegap-xapkreader#20]
Hi there.

When we upload a new APK to production (e.g. Version code: 10 and version number: 1.0.10) can we reuse the previous expansion file (version 1.0.9)? Or do we need to reupload the same expansion file each time? 

If we can reuse the previous version expansion file, then should I be using main_version = 9? Currently I keep increment both my app version number as well as the expansion version and upload the same expansion file every time. 

`<?xml version='1.0' encoding='utf-8'?>
<resources xmlns:android="http://schemas.android.com/apk/res/android">
    <string name="downloading_assets">Downloading assets… (80.08 MB)</string>
    <string name="preparing_assets">Preparing assets…</string>
    <string name="download_failed">Download failed. Please restart app.</string>
    <string name="error">Error</string>
    <string name="close">Close</string>
    <integer name="main_version">10</integer>
    <integer name="patch_version">10</integer>
    <integer name="file_size">83973142</integer>
</resources>`

Last question: If we reuse previous version expansion file, does that mean that the users dont need to re-download the full 80mb expansion file each time we roll out an update?

Thanks so much! 
Asfahaan

**Comment by lumierephoto** [comment]
Good question! I keep uploading new ones, because the version number changes and I want it to match my APK version.

**Comment by moregamers** [comment]
yes, you can reuse a previous APK.

**Comment by psyon** [comment]
Yes, you can use a previously uploaded expansion file.  When you upload a new SDK it will ask you which one you want to use.  Pick the one that matches the version you specified when installing the plugin.
